src/include/miopen/tensor.hpp

```diff
@@ -124,45 +124,56 @@ std::ptrdiff_t integer_division_ceil(X x, Y y)
 struct TensorDescriptor : miopenTensorDescriptor
 {
     TensorDescriptor();
-    TensorDescriptor(miopenDataType_t t, std::initializer_list<std::size_t> plens);
+
+    // This constructor is only used in test/tensor_holder.hpp
+    // clang-format off
+    [[deprecated("Use constructor with lengths instead")]]
```

Comment:
[anchor]

Reply:
@JehandadKhan Could you please clarify if this is good or bad? :smile:

Reply:
I think this is good, I put the anchor to share it with Jun

Reply:
@averinevg This is good practice, I usually add `:anchor:` which is rendered as :anchor: